app/templates/components/github-apps-repository.hbs

```diff
@@ -4,9 +4,43 @@
   {{name}}
 {{/link-to}}
 
-{{#unless repository.active_on_org}}
+{{#if repository.active_on_org}}
+  {{#if repository.permissions.migrate}}
+    {{#if repository.migrationStatus}}
+      <div
+        data-test-migration-status={{repository.migrationStatus}}
+        class="migration-status {{repository.migrationStatus}}"
+      >
+      {{#if (eq repository.migrationStatus "migrating")}}
+        {{loading-indicator
+          data-test-repository-migration-loading-animation
+          inline=true
+        }}
+      {{/if}}
+      {{#if (eq repository.migrationStatus "success")}}
+        <span>{{svg-jar "stage-passed" class="icon"}}</span>
+      {{/if}}
+      {{#if (eq repository.migrationStatus "failed")}}
+        <span>{{svg-jar "stage-errored" class="icon"}}</span>
+      {{/if}}
+      </div>
+    {{else}}
+      <a
+        data-test-migrate-repository-button
+        href="#"
```

Comment:
In some browsers links with `herf="#"` make page jump to top. It may be safer to remove `href` and add `pointer` class.